src/key.h

```diff
@@ -170,6 +170,23 @@ struct CExtKey {
     bool Derive(CExtKey& out, unsigned int nChild) const;
     CExtPubKey Neuter() const;
     void SetMaster(const unsigned char* seed, unsigned int nSeedLen);
+     template <typename Stream>
+    void Serialize(Stream& s, int nType, int nVersion) const
+    {
+        unsigned int len = 74;
```

Comment:
nit: What's the meaning of 74 here? Is it the same in the two following arrays init?

Reply:
the meaning of 74 here is the BIP32 EXTKEY SIZE ,in dash's pr it was hardcoded in ,should i follow bitcoin 's pr and delclare a var named BIP32_EXTKEY_SIZE  and reference that instead?

Reply:
looks like that var already is defined,might aswell reference that instead of hardcoding it,thanks for pointing it out